packages/enzyme/src/Utils.js

```diff
@@ -7,8 +7,47 @@ import configuration from './configuration';
 import validateAdapter from './validateAdapter';
 import { childrenOfNode } from './RSTTraversal';
 
+const hasSymbol = typeof Symbol === 'function' && Symbol.for;
+
 export const ITERATOR_SYMBOL = typeof Symbol === 'function' && Symbol.iterator;
 
+// TODO use react-is when a version is released with no peer dependency on React
```

Comment:
The latest react-is release has your change in it and does not have react as peer dependency anymore. Could this be updated?

Reply:
a) `react-is` must never be used in new places outside of a react adapter, since non-react adapters exist. b) `react-is` does not work on react 0.13.

So, all of these things need to be exposed via the adapter interface - instead of being part of enzyme.

Reply:
I'm not sure what's actionable here. It' seems like you are saying we should update all adapters to provide a `typeOf` api?

Reply:
Yes, i think that might be what's required :-/

if so, let's do that in a separate PR, so that when this one is rebased, it can only include the diffs to typeOf?